docs/baseline_results/object_detection_poisoning_results.md

```diff
@@ -93,6 +93,80 @@ See [the paper](https://arxiv.org/pdf/2205.14497.pdf) for a detailed description
 | 30 |  0.493 | 0.447 | 0.494 | 0.447 | 0.077 | 0.230 | 0.015 |
 
 
+## Skull trigger
```

Comment:
Maybe link to the triggers within armory here such as https://github.com/twosixlabs/armory/blob/2539af164d9c78e7d8abdc8b405a1700a5f594eb/armory/utils/triggers/skull.png
```suggestion
## [Skull trigger](../../armory/utils/triggers/skull.png)
```
or mention where they are found (i.e. armory/utils/triggers)

Reply:
I put a link at the top.